### DISCUSSION TITLE
IDH High Disk Utilization

### DISCUSSION THREAD

**Discussion by M4jx** [Security-Onion-Solutions/securityonion#12622] [Unsupported Versions] [answered]
I have observed that the IDH node have high disk utilization. Is that expected?

Version: 2.4.50
Memory: 1GB
CPU: 2 cores
Disk: 100GB

Here is a screenshot from Proxmox statistics of the IDH VM.

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/25796885/31777900-8d1c-4e62-9990-790fec24abf0)

**Comment by reyesj2** [comment, marked as answer]
That looks like its roughly every 15 minutes which lines up with salt highstates. So the system checking that its in the correct configuration according to the manager.
